### PR TITLE
Nep17

### PR DESCRIPTION
**Pull request by chenzhitong**
close https://github.com/neo-project/neo-modules/issues/405

### CODE REVIEW

**Comment by chenzhitong** [comment]
TODO: Awaiting manifest data structure changes in the neo project, and fixe UT.

File: src/RpcClient/Models/RpcNep5TokenInfo.cs [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 namespace Neo.Network.RPC.Models
 {
-    public class RpcNep5TokenInfo
+    public class RpcNep17TokenInfo
     {
-        public string Name { get; set; }
+        public string Neme { get; set; }
```
Comment (vncoelho) [inline src/RpcClient/Models/RpcNep5TokenInfo.cs:7]:
```suggestion
        public string Name { get; set; }
```

File: src/RpcClient/Nep17API.cs [hunk 2]
```diff
@@ -85,40 +85,44 @@ public async Task<BigInteger> TotalSupplyAsync(UInt160 scriptHash)
...
             var stack = result.Stack;
 
-            return new RpcNep5TokenInfo
+            return new RpcNep17TokenInfo
             {
-                Name = stack[0].GetString(),
-                Symbol = stack[1].GetString(),
-                Decimals = (byte)stack[2].GetInteger(),
-                TotalSupply = stack[3].GetInteger()
+                Neme = string.Empty,
```
Comment (vncoelho) [inline src/RpcClient/Nep17API.cs:104]:
```suggestion
                Name = string.Empty,
```

File: README.md [hunk 3]
```diff
@@ -65,12 +65,12 @@ You can also use `RocksDBStore` in the NEO system by modifying the default stora
 ### RpcServer
 Plugin for hosting a RpcServer on the neo-node, being able to disable specific calls.
 
-### RpcNep5Tracker
-Plugin that enables NEP5 tracking using LevelDB.
+### Rpc17Tracker
```
Comment (superboyiii) [inline README.md:68]:
```suggestion
### RpcNep17Tracker
```

File: src/RpcClient/Models/RpcNep5Balances.cs [hunk 4]
```diff
@@ -6,11 +6,11 @@
 
 namespace Neo.Network.RPC.Models
 {
-    public class RpcNep5Balances
+    public class RpcNep17Balances
```
Comment (shargon) [inline src/RpcClient/Models/RpcNep5Balances.cs:9]: File names with `Nep5`

**Review by shargon — CHANGES_REQUESTED** [review]

**Comment by chenzhitong** [comment]
`TestGetTokenInfo()` How do I fix this unit test? 
image

**Comment by superboyiii** [comment]
@shargon Works well on mine. Need your review again.

**Review by superboyiii — APPROVED** [review]

File: src/RpcClient/RpcClient.cs [hunk 5]
```diff
@@ -192,7 +192,7 @@ public async Task<RpcBlockHeader> GetBlockHeaderAsync(string hashOrIndex)
         /// </summary>
         public async Task<ContractState> GetContractStateAsync(string hash)
         {
-            var result = await RpcSendAsync(GetRpcName(), hash).ConfigureAwait(false);
+            var result = await RpcSendAsync(GetRpcName(), hash).ConfigureAwait(true);
```
Comment (shargon) [inline src/RpcClient/RpcClient.cs:195]: This change it's not related to the PR, mistake?

File: tests/Neo.Network.RPC.Tests/UT_WalletAPI.cs [hunk 6]
```diff
@@ -127,6 +127,21 @@ public async Task TestTransferfromMultiSigAccount()
 
             var tranaction = await walletAPI.TransferAsync(NativeContract.GAS.Hash, 1, new[] { keyPair1.PublicKey }, new[] { keyPair1 }, UInt160.Zero, NativeContract.GAS.Factor * 100);
             Assert.AreEqual(testScript.ToHexString(), tranaction.Script.ToHexString());
+
+            try
+            {
+                tranaction = await walletAPI.TransferAsync(NativeContract.GAS.Hash, 2, new[] { keyPair1.PublicKey }, new[] { keyPair1 }, UInt160.Zero, NativeContract.GAS.Factor * 100);
```
Comment (shargon) [inline tests/Neo.Network.RPC.Tests/UT_WalletAPI.cs:133]: `transaction` this test was wrong designed because it has two possible paths. Pass, and Fail.

**Review by shargon — APPROVED** [review]

**Pull request merged**